app/services/change_schedule.rb

```diff
@@ -143,11 +143,20 @@ def update_npq_application_cohort!
     participant_profile.npq_application.update!(cohort: new_schedule.cohort)
   end
 
+  def update_funded_place!
+    return unless FeatureFlag.active?(:npq_capping)
+    return unless participant_profile.npq_application.cohort.start_year >= 2024
```

Comment:
could we drive this via the contracts (see comment from accept pr around how to query those) happy to have a chat about that

Reply:
maybe we can also check if they are moving cohorts so we don't unset an existing value for 2024 moving between schedules?